graphql/schema.graphql

```diff
@@ -78,26 +88,29 @@ type Query {
   getPaymentTerms: [PaymentTerm]
     @cacheControl(scope: PUBLIC, maxAge: SHORT)
     @auditAccess
-
+    @checkAccessWithFeatureFlag
   getOrganizationsByEmail(email: String): [B2BOrganization]
     @checkUserAccess
     @cacheControl(scope: PRIVATE)
     @auditAccess
```

Comment:
`getOrganizationsByEmail` is on https://vtex-dev.atlassian.net/browse/B2BTEAM-1285 but we haven't add the `@checkAccessWithFeatureFlag` for it

Reply:
It is not necessary, because there is already with directive `@checkUserAccess`.